app/src/userpages/components/Header/Tab/index.jsx

```diff
@@ -0,0 +1,26 @@
+// @flow
+
+import React, { type Node } from 'react'
+import cx from 'classnames'
+import { Link, type Location, withRouter } from 'react-router-dom'
+import styles from './tab.pcss'
+
+type Props = {
+    children: Node,
+    className?: ?string,
+    location: Location,
+    to: string,
+}
+
+const Tab = withRouter(({ className, to, location: { pathname }, children }: Props) => (
+    <Link
+        className={cx(styles.root, className, {
+            [styles.active]: pathname.includes(to),
```

Comment:
`pathname.includes` seems very brittle to me, `pathname.startsWith` would maybe be better?

Also could `NavLink` do this for us?

https://reacttraining.com/react-router/web/api/NavLink

Reply:
Also note this potential gotcha: 

![image](https://user-images.githubusercontent.com/43438/58532464-3a45f500-8218-11e9-8361-d04a65222f32.png)

https://reacttraining.com/react-router/web/api/withRouter